.pre-commit-config.yaml

```diff
@@ -10,22 +10,17 @@ repos:
   hooks:
   - id: trailing-whitespace
     exclude: tests/foreman/data/
-  - id: end-of-file-fixer
   - id: check-yaml
   - id: debug-statements
```

Comment:
Can you also run `pre-commit autoupdate; pre-commit run -a` to update the available pre-commit hooks and code changes?

Reply:
I would like to do it in a separate PR.

Reply:
#12650 created